scripts/verify-runtime-themes.ts

```
@@ -0,0 +1,281 @@
+const firstEntry = firstSnapshot.find(entry => entry.name === 'probe:valid')
+check('valid registration appears in the host snapshot', firstEntry !== undefined)
+check('registration normalizes and sanitizes display metadata',
+  firstEntry?.displayName === 'Valid Runtime Theme' && firstEntry?.name === 'probe:valid')
+check('registration snapshot and colors are frozen',
+  firstEntry !== undefined && Object.isFrozen(firstEntry) && Object.isFrozen(firstEntry.colors))
+check('snapshot stays stable without a mutation', host.getSnapshot() === firstSnapshot)
+check('host resolver returns the registered palette', host.resolve('probe:valid')?.claude === '#123456')
+check('getTheme consults the runtime resolver', getTheme('probe:valid').claude === '#123456')
+check('runtime availability includes the registered name', isThemeAvailable('probe:valid'))
+check('static availability remains static-only', !isStaticThemeAvailable('probe:valid'))
+check('host resolver returns a stable palette identity', host.resolve('probe:valid') === host.resolve('probe:valid'))
```

Biome (2.5.7)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)